src/move.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-console */
+'use strict';
+
+const { existsSync } = require('fs');
+const fs = require('fs/promises');
+const path = require('path');
+
+const moveFile = async(fromPath, toPath) => {
+  if (!existsSync(fromPath)) {
+    console.error(`Source file "${fromPath}" does not exist!`);
+
+    return;
+  }
+
+  try {
+    let newPath = toPath;
+
+    if (existsSync(newPath)) {
+      newPath = path.join(toPath, '/', path.basename(fromPath));
+
+      await fs.rename(fromPath, newPath);
+    } else {
+      await fs.rename(fromPath, newPath);
+    }
+
+    console.log(`Successfully moved from "${fromPath}" to "${newPath}"`);
+  } catch (err) {
+    console.error(err);
```

Comment:
You shouldn't throw an original error to a user. Parse useful info, remove any sensitive data (in this case there is no any), and write a user-friendly error.
![image](https://user-images.githubusercontent.com/45945732/233973833-6d268223-1860-447b-bf54-7c030623b4d6.png)